src/app/components/url-validation/index.html

```diff
@@ -6,17 +6,24 @@
 >
 
   <sky-demo-page-summary>
-    The URL validation module allows the user to validate URL format.
+    The URL validation module validates the format of URLs in input fields.
   </sky-demo-page-summary>
 
-  <sky-demo-page-properties sectionHeading="Directive properties">
+  <sky-demo-page-properties sectionHeading="URL validation input directive properties">
     <sky-demo-page-property
       propertyName="skyUrlValidation"
     >
-    When this attribute is placed on an input element, the input with validation properties is created. The error message will display if the input is not in the correct format. This directive will use <sky-code>NgModel</sky-code> for binding data.
+      Creates an input that validates URLs. Place this attribute on an <sky-code>input</sky-code> element. If users enters values that are not valid URLs, an error message appears. This directive uses <sky-code>NgModel</sky-code> to bind data.
     </sky-demo-page-property>
   </sky-demo-page-properties>
 
+  <sky-demo-page-properties sectionHeading="URL validator function properties">
+    <sky-demo-page-property propertyName="SkyValidators.url">
+      Creates an input that validates URLs in reactive forms. Add this validator directly to the form control model in the component class. If users enter values that are not valid URLs, an error message appears. Since this is a sync validator, it returns a set of validation errors or <sky-code>undefined</sky-code> immediately when users enter values.
```

Comment:
Same thoughts about the wording here as in the email one regarding the input creation and the errors.

Reply:
Will reword and repush.

Reply:
John will reword and repush :)